Source/WebCore/platform/graphics/avfoundation/cf/MediaPlayerPrivateAVFoundationCF.cpp

```diff
@@ -1426,6 +1428,10 @@ AVFWrapper::~AVFWrapper()
     m_avPlayerItem = 0;
     m_timeObserver = 0;
     m_avPlayer = 0;
+
+    ASSERT(!m_seekCompletion);
+    if (m_seekCompletion)
+        (*m_seekCompletion)(MediaPlayerEnums::SeekResult::Cancelled);
```

Comment:
shouldn't m_seekCompletion be cleared after it's been called?

Reply:
Should the `MediaPlayerPrivateAVFoundation` destructor also call `m_seekCompletion` if it is non-null?